test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
```

Comment:
Is there even a remote chance that `diffFromNow` will be `0`? I imagine the answer is "no".

(Hey, look at me! I'm making myself ask a question that I might normally avoid asking out of fear it will make me look ignorant or foolish. Everyone should try this! And now I'm masking my insecurity with self-congratulation! You now know more about my inner psychological workings than you ever wanted to! Life is amazing!!)